cmsl1t/plotting/resolution_vs_X.py

```diff
@@ -45,18 +48,36 @@ def create_histograms(self,
         self.filename_format = name
 
     def fill(self, pileup, versus, offline, online):
-        difference = self.resolution_method(online, offline)
+        difference = self.resolution_method(online, offline) + 1.
```

Comment:
@professor-calculus Could you please remind me what the `+1` is for?

Reply:
For Olivier’s resolution study they wanted to define resolution as online/offline rather than (online-offline)/offline — was easiest to add 1 since it’s for a specific study rather than a general thing